src/app/components/core/south/south-service-modal/south-service-modal.component.ts

```diff
@@ -195,9 +195,20 @@ export class SouthServiceModalComponent implements OnInit, OnChanges {
    * @param changedConfig changed configuration of a selected plugin
    */
   getChangedConfig(changedConfig) {
```

Comment:
This method should not call in the case of clicking `Show/Hide Advanced Config` in order to remove the error in console `ERROR TypeError: Cannot read property 'toUpperCase' of undefined`

Reply:
@monika-Dianomic  this seems to get fixed with

```
    if (isEmpty(changedConfig)) {
      return;
    }
```